build.sbt

```diff
@@ -219,10 +240,17 @@ val commonSettings = Def
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint:unchecked"),
     Compile / doc / javacOptions := Seq("-source", "1.8"),
     excludeDependencies ++= Seq(
-      "org.apache.beam" % "beam-sdks-java-io-kafka"
+      "org.apache.beam" % "beam-sdks-java-io-kafka",
+      "org.slf4j" % "slf4j-log4j12",
+      "org.slf4j" % "slf4j-reload4j"
     ),
     resolvers ++= Resolver.sonatypeOssRepos("public"),
-    Test / javaOptions += "-Dscio.ignoreVersionWarning=true",
+    Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat,
+    Test / javaOptions ++= Seq(
+      "-Dscio.ignoreVersionWarning=true",
+      "-Dorg.slf4j.simpleLogger.defaultLogLevel=info",
+      "-Dorg.slf4j.simpleLogger.logFile=scio.log"
```

Comment:
log config for forked tests

Reply:
Double checked the `sbt` doc and you are actually right. all options from the main JVM should be propagated. Will try to remove the redundant ones